### PR TITLE
Fixed couple of QoS test cases for Z9332 platform

### PR DESCRIPTION
**Pull request by vmittal-msft**
### Description of PR

Summary:
Fixes # (issue)
Fixes couple of QoS test cases for Z9332 on T0 and t1 topology

### Type of change
Updated QoS test parameters as well as test logic .

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Fix QoS test cases for Z9332 platform.

#### How did you do it?
Updated test params and test logic for Z9332.

#### How did you verify/test it?
Verified these test cases on HWSKU DellEMC-Z9332f-O32 and DellEMC-Z9332f-M-O16C64

### CODE REVIEW

File: tests/saitests/sai_qos_tests.py [hunk 1]
```diff
@@ -1535,6 +1536,10 @@ def runTest(self):
         )
         print >> sys.stderr, "actual dst_port_id: {}".format(dst_port_id)
 
+        # For TH3, some packets stay in egress memory and doesn't show up in shared buffer or leakout
+        if 'pkts_num_egr_mem' in self.test_params.keys():
+            pkts_num_egr_mem = int(self.test_params['pkts_num_egr_mem'])
```
Comment (neethajohn) [inline tests/saitests/sai_qos_tests.py:1541]: don't see this var used in this test

File: tests/saitests/sai_qos_tests.py [hunk 2]
```diff
@@ -1509,6 +1509,7 @@ def runTest(self):
         limit = int(self.test_params['limit'])
         pkts_num_leak_out = int(self.test_params['pkts_num_leak_out'])
         topo = self.test_params['topo']
+        hwsku = self.test_params['hwsku']
```
Comment (neethajohn) [inline tests/saitests/sai_qos_tests.py:1512]: don't see this var used in this test

File: tests/saitests/sai_qos_tests.py (hunk 1 above)
Comment (vmittal-msft) [reply tests/saitests/sai_qos_tests.py]: Fixed

File: tests/saitests/sai_qos_tests.py (hunk 2 above)
Comment (vmittal-msft) [reply tests/saitests/sai_qos_tests.py]: Fixed

**Review by neethajohn — APPROVED** [review]

**Pull request merged**